docs/nimcache.md

```diff
@@ -77,13 +83,13 @@ spec:
       volumeAccessMode: ReadWriteOnce
```

Comment:
On line 74, can you s/gpu/gpus/?  Sorry for that.

Reply:
@mikemckiernan updated to gpus

NOTE: looks like we're missing it as part of original doc